src/configuration-factory.ts

```diff
@@ -5,6 +5,7 @@ import {
   MemoryOnlyConfigurationStore,
   MemoryStore,
 } from '@eppo/js-client-sdk-common';
+import { PrecomputedFlag } from '@eppo/js-client-sdk-common/dist/interfaces';
```

Comment:
TODO: update and publish `js-client-sdk-common` so that this is included in the exports